content/influxdb3/cloud-dedicated/admin/identify-version.md

```diff
@@ -4,8 +4,8 @@ description: Learn how to identify your InfluxDB Cloud Dedicated cluster through
 menu:
   influxdb3_cloud_dedicated:
     name: Identify version
-    parent: Administer InfluxDB
-weight: 10
+    parent: Administer InfluxDB Cloud
```

Comment:
In this portion of the documentation the section labeled `Administer InfluxDB` is `Administer InfluxDB Cloud`. This has left the 'identify version' subheading that appears to avoid confusion in each of the InfluxDB3 products in its own subheading.

Before Fix:
<img width="260" height="537" alt="Screenshot 2025-11-21 at 12 26 00 PM" src="https://github.com/user-attachments/assets/287ed75e-d822-450d-9fb3-3e30e5a600ce" />


After Fix:
<img width="260" height="537" alt="Screenshot 2025-11-21 at 12 26 13 PM" src="https://github.com/user-attachments/assets/8cb3d97f-48d3-43c7-9f89-4fb567562108" />